python/smells/dict-modify-iterating.yaml

```diff
@@ -10,6 +10,12 @@ rules:
               for $KEY in $DICT.keys():
                   ...
                   del $DICT[$KEY]
+          - pattern: |
+              for $KEY, $VALUE in $DICT.items():
+                  del $DICT[$KEY]
+          - pattern: |
+              for $KEY in $DICT.keys():
+                  del $DICT[$KEY]
```

Comment:
@ulziibay did you accidentally commit this?

Reply:
No, they were failing CI

Reply:
What is going on here? This fixes the CI @ievans